src/plugins/googledrive/server/controllers/operation-controller.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+module.exports = ({ strapi }) => ({
+  get: async (ctx) => {
+    try {
+      ctx.body = await strapi
+        .plugin('googledrive')
+        .service('operationService')
+        .get(ctx);
+    } catch (err) {
```

Comment:
@jeyanthi-sm will these errors get logged? If not (or if we're unsure) can we add a `console.log` here with a concise message and the error?

Reply:
I think this will return the error to the caller. I think we should return a 500 internal server error with a generic message. Usually it's not recommended to return internal error directly to users.